noir-projects/noir-contracts/scripts/bootstraph_just_one_contract.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+set -eu
+
+cd "$(dirname "$0")"
+
+# Call example:
+# `./scripts/bootstraph_just_one_contract.sh nft_contract NFT`
```

Comment:
New script to just compile one of the contracts, and do all the build stuff that makes it work with typescript tests.

Reply:
omg i love this

Reply:
Lovely!

But there is a typo in the command name: `bootstrap**h**_just_one_contract.sh`